javaschool-backend/src/main/java/com/nearsoft/javaschoolbackend/controller/PackageInfoController.java

```diff
@@ -0,0 +1,30 @@
+package com.nearsoft.javaschoolbackend.controller;
+
+import com.nearsoft.javaschoolbackend.service.PackageInfoService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/shippingAPI/packageInfo")
```

Comment:
Use hyphens (-) to improve the readability of URIs.
[https://restfulapi.net/resource-naming/](url)